java/ql/lib/semmle/code/java/security/Xxe.qll

```diff
@@ -0,0 +1,24 @@
+/** Provides classes to reason about XML eXternal Entity (XXE) vulnerabilities. */
+
+import java
+import semmle.code.java.security.XmlParsers
```

Comment:
unused import

Reply:
Or, I guess it's probably used in `java/ql/lib/semmle/code/java/security/XxeQuery.qll`, so move it there?